### PR TITLE
[Localization] New translations batch locales (Move-disabling effects, abilities triggers, error messages)

### PR DESCRIPTION
**Pull request by Adri1**
## What are the changes the user will see?
Added missing translations to locales (Move-disabling effects, abilities triggers, error messages)
**I added English placeholders to facilitate suggestions, but if a language never provide any translation, I'll remove these placeholders before merge.**

## Why am I making these changes?
Having them translated

## Checklist
- [X] **I'm using `beta` as my base branch**
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?

### CODE REVIEW

File: src/locales/de/battle.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
   "moveNotImplemented": "{{moveName}} ist noch nicht implementiert und kann nicht ausgewählt werden.",
   "moveNoPP": "Es sind keine AP für diese Attacke mehr übrig!",
   "moveDisabled": "{{moveName}} ist deaktiviert!",
+  "disableInterruptedMove": "{{pokemonNameWithAffix}}'s {{moveName}}\nis disabled!",
```
Comment (CodeTappert) [inline src/locales/de/battle.json:47]:
```suggestion
  "disableInterruptedMove": "{{moveName}} von {{pokemonNameWithAffix}} ist blockiert!",
```

File: src/locales/de/battler-tags.json [hunk 2]
```diff
@@ -67,5 +67,7 @@
   "saltCuredLapse": "{{pokemonNameWithAffix}} wurde durch {{moveName}} verletzt!",
   "cursedOnAdd": "{{pokemonNameWithAffix}} nimmt einen Teil seiner KP und legt einen Fluch auf {{pokemonName}}!",
   "cursedLapse": "{{pokemonNameWithAffix}} wurde durch den Fluch verletzt!",
-  "stockpilingOnAdd": "{{pokemonNameWithAffix}} hortet {{stockpiledCount}}!"
-}
\ No newline at end of file
+  "stockpilingOnAdd": "{{pokemonNameWithAffix}} hortet {{stockpiledCount}}!",
+  "disabledOnAdd": "{{pokemonNameWithAffix}}'s {{moveName}}\nwas disabled!",
+  "disabledLapse": "{{pokemonNameWithAffix}}'s {{moveName}}\nis no longer disabled."
```
Comment (CodeTappert) [inline src/locales/de/battler-tags.json:72]:
```suggestion
  "stockpilingOnAdd": "{{pokemonNameWithAffix}} hortet {{stockpiledCount}}!",
  "disabledOnAdd": " {{moveName}}  von {{pokemonNameWithAffix}} wurde blockiert!",
  "disabledLapse": "{{moveName}} von {{pokemonNameWithAffix}} ist nicht länger blockiert!"
```

File: src/locales/ko/battle.json [hunk 3]
```diff
@@ -44,6 +44,7 @@
   "moveNotImplemented": "{{moveName}}[[는]] 아직 구현되지 않아 사용할 수 없다…",
   "moveNoPP": "기술의 남은 포인트가 없다!",
   "moveDisabled": "{{moveName}}[[를]] 쓸 수 없다!",
+  "disableInterruptedMove": "{{pokemonNameWithAffix}}'s {{moveName}}\nis disabled!",
```
Comment (sodaMelon) [inline src/locales/ko/battle.json:47]:
```suggestion
  "disableInterruptedMove": "{{pokemonNameWithAffix}}의 {{moveName}}[[는]]\n 사용할 수 없다.",
```

File: src/locales/ko/battler-tags.json [hunk 4]
```diff
@@ -67,5 +67,7 @@
   "saltCuredLapse": "{{pokemonNameWithAffix}}[[는]] 소금절이의\n데미지를 입고 있다.",
   "cursedOnAdd": "{{pokemonNameWithAffix}}[[는]] 자신의 체력을 깎아서\n{{pokemonName}}에게 저주를 걸었다!",
   "cursedLapse": "{{pokemonNameWithAffix}}[[는]]\n저주받고 있다!",
-  "stockpilingOnAdd": "{{pokemonNameWithAffix}}[[는]]\n{{stockpiledCount}}개 비축했다!"
-}
\ No newline at end of file
+  "stockpilingOnAdd": "{{pokemonNameWithAffix}}[[는]]\n{{stockpiledCount}}개 비축했다!",
+  "disabledOnAdd": "{{pokemonNameWithAffix}}'s {{moveName}}\nwas disabled!",
+  "disabledLapse": "{{pokemonNameWithAffix}}'s {{moveName}}\nis no longer disabled."
```
Comment (sodaMelon) [inline src/locales/ko/battler-tags.json:72]:
```suggestion
  "disabledOnAdd": "{{pokemonNameWithAffix}}의 {{moveName}}[[는]]\n사용할 수 없었다!",
  "disabledLapse": "{{pokemonNameWithAffix}}의 {{moveName}}[[는]]\n이제 사용할 수 있다."
```

**Review by sodaMelon — COMMENTED** [review]
added korean localization

File: src/locales/it/battle.json [hunk 5]
```diff
@@ -44,6 +44,7 @@
   "moveNotImplemented": "{{moveName}} non è ancora implementata e non può essere selezionata.",
   "moveNoPP": "Non ci sono PP rimanenti\nper questa mossa!",
   "moveDisabled": "{{moveName}} è disabilitata!",
+  "disableInterruptedMove": "{{pokemonNameWithAffix}}'s {{moveName}}\nis disabled!",
```
Comment (NicusPulcis) [inline src/locales/it/battle.json:47]:
```suggestion
  "disableInterruptedMove": "La mossa {{moveName}} di\n{{pokemonNameWithAffix}} è bloccata!",
```

**Review by NicusPulcis — DISMISSED** [review]
italian localization done

File: src/locales/de/ability-trigger.json [hunk 6]
```diff
@@ -12,6 +12,7 @@
   "typeImmunityHeal": "{{abilityName}} von {{pokemonNameWithAffix}} füllte einige KP auf!",
   "nonSuperEffectiveImmunity": "{{pokemonNameWithAffix}} vermeidet Schaden mit {{abilityName}}!",
   "disguiseAvoidedDamage": "Die Tarnung von {{pokemonNameWithAffix}} ist aufgeflogen!!",
+  "fullHpResistType": "{{pokemonNameWithAffix}} made its shell gleam!\nIt's distorting type matchups!",
```
Comment (CodeTappert) [inline src/locales/de/ability-trigger.json:15]:
```suggestion
  "fullHpResistType": "Der Panzer von {{pokemonNameWithAffix}} funkelt und verzerrt
die Wechselwirkungen zwischen den Typen!",
```

File: src/locales/de/ability-trigger.json [hunk 7]
```diff
@@ -51,6 +52,7 @@
   "postSummonTeravolt": "{{pokemonNameWithAffix}} strahlt eine knisternde Aura aus!",
   "postSummonDarkAura": "{{pokemonNameWithAffix}} strahlt eine dunkle Aura aus!",
   "postSummonFairyAura": "{{pokemonNameWithAffix}} strahlt eine Feenaura aus!",
+  "postSummonAuraBreak": "{{pokemonNameWithAffix}} reversed all other Pokémon's auras!",
```
Comment (CodeTappert) [inline src/locales/de/ability-trigger.json:55]:
```suggestion
  "postSummonAuraBreak": "{{pokemonNameWithAffix}} kehrt die Wirkung aller Aura-Fähigkeiten um!",
```

File: src/locales/de/menu.json [hunk 8]
```diff
@@ -51,5 +51,7 @@
   "renamePokemon": "Pokémon umbennenen",
   "rename": "Umbenennen",
   "nickname": "Spitzname",
-  "errorServerDown": "Ups! Es gab einen Fehler beim Versuch\nden Server zu kontaktieren\nLasse dieses Fenster offen\nDu wirst automatisch neu verbunden."
-}
\ No newline at end of file
+  "errorServerDown": "Ups! Es gab einen Fehler beim Versuch\nden Server zu kontaktieren\nLasse dieses Fenster offen\nDu wirst automatisch neu verbunden.",
+  "noSaves": "You don't have any save files on record!",
+  "tooManySaves": "You have too many save files on record!"
```
Comment (CodeTappert) [inline src/locales/de/menu.json:56]:
```suggestion
  "noSaves": "Du hast keine gespeicherten Dateien!",
  "tooManySaves": "Du hast zu viele gespeicherte Dateien!"

```

File: src/locales/ko/ability-trigger.json [hunk 9]
```diff
@@ -12,6 +12,7 @@
   "blockItemTheft": "{{pokemonNameWithAffix}}의 {{abilityName}}에 의해\n도구를 빼앗기지 않는다!",
   "typeImmunityHeal": "{{pokemonNameWithAffix}}[[는]]\n{{abilityName}}[[로]] 체력이 회복되었다!",
   "nonSuperEffectiveImmunity": "{{pokemonNameWithAffix}}[[는]] {{abilityName}} 때문에\n데미지를 입지 않는다!",
+  "fullHpResistType": "{{pokemonNameWithAffix}} made its shell gleam!\nIt's distorting type matchups!",
```
Comment (KimJeongSun) [inline src/locales/ko/ability-trigger.json:15]:
```suggestion
  "fullHpResistType": "{{pokemonNameWithAffix}}[[는]] 등껍질을 빛나게 하여\n타입 상성을 왜곡시켰다!!",
```
image

File: src/locales/ko/ability-trigger.json [hunk 10]
```diff
@@ -51,6 +52,7 @@
   "postSummonTeravolt": "{{pokemonNameWithAffix}}[[는]]\n세차게 튀는 오라를 발산하고 있다!",
   "postSummonDarkAura": "{{pokemonNameWithAffix}}[[는]]\n다크오라를 발산하고 있다!",
   "postSummonFairyAura": "{{pokemonNameWithAffix}}[[는]]\n페어리오라를 발산하고 있다!",
+  "postSummonAuraBreak": "{{pokemonNameWithAffix}} reversed all other Pokémon's auras!",
```
Comment (KimJeongSun) [inline src/locales/ko/ability-trigger.json:55]:
```suggestion
  "postSummonAuraBreak": "{{pokemonNameWithAffix}}[[는]]\n모든 오라를 제압한다!",
```
image

File: src/locales/ko/menu.json [hunk 11]
```diff
@@ -51,5 +51,7 @@
   "renamePokemon": "포켓몬의 닉네임은?",
   "rename": "닉네임 바꾸기",
   "nickname": "닉네임",
-  "errorServerDown": "서버 연결 중 문제가 발생했습니다.\n\n이 창을 종료하지 않고 두면,\n게임은 자동으로 재접속됩니다."
+  "errorServerDown": "서버 연결 중 문제가 발생했습니다.\n\n이 창을 종료하지 않고 두면,\n게임은 자동으로 재접속됩니다.",
+  "noSaves": "You don't have any save files on record!",
```
Comment (KimJeongSun) [inline src/locales/ko/menu.json:55]:
```suggestion
  "noSaves": "기기에 세이브 파일이 없습니다!",
```

File: src/locales/ko/menu.json [hunk 12]
```diff
@@ -51,5 +51,7 @@
...
+  "tooManySaves": "You have too many save files on record!"
```
Comment (KimJeongSun) [inline src/locales/ko/menu.json:56]:
```suggestion
  "tooManySaves": "기기에 세이브 파일이 너무 많습니다!"
```

**Review by KimJeongSun — CHANGES_REQUESTED** [review]

File: src/locales/es/battler-tags.json [hunk 13]
```diff
@@ -67,5 +67,7 @@
   "saltCuredLapse": "¡{{moveName}} ha herido a {{pokemonNameWithAffix}}!",
   "cursedOnAdd": "¡{{pokemonNameWithAffix}} sacrifica algunos PS y maldice a {{pokemonName}}!",
   "cursedLapse": "¡{{pokemonNameWithAffix}} es víctima de una maldición!",
-  "stockpilingOnAdd": "¡{{pokemonNameWithAffix}} ha reservado energía por {{stockpiledCount}}ª vez!"
+  "stockpilingOnAdd": "¡{{pokemonNameWithAffix}} ha reservado energía por {{stockpiledCount}}ª vez!",
+  "disabledOnAdd": "{{pokemonNameWithAffix}}'s {{moveName}}\nwas disabled!",
```
Comment (DanStevensonx) [inline src/locales/es/battler-tags.json:71]: "disabledOnAdd": "¡Se ha anulado el movimiento {{moveName}}\nde {{pokemonNameWithAffix}}!",

File: src/locales/es/battler-tags.json [hunk 14]
```diff
@@ -67,5 +67,7 @@
...
+  "disabledLapse": "{{pokemonNameWithAffix}}'s {{moveName}}\nis no longer disabled."
```
Comment (DanStevensonx) [inline src/locales/es/battler-tags.json:72]: "disabledLapse": "¡El movimiento {{moveName}} de {{pokemonNameWithAffix}} \n ya no está anulado!"

File: src/locales/es/battle.json [hunk 15]
```diff
@@ -42,6 +42,7 @@
   "moveNotImplemented": "{{moveName}} aún no está implementado y no se puede seleccionar.",
   "moveNoPP": "¡No hay suficientes PP\npara este movimiento!",
   "moveDisabled": "!No puede usar {{moveName}} porque ha sido anulado!",
+  "disableInterruptedMove": "{{pokemonNameWithAffix}}'s {{moveName}}\nis disabled!",
```
Comment (DanStevensonx) [inline src/locales/es/battle.json:45]: "disableInterruptedMove": "¡Se ha anulado el movimiento {{moveName}}\nde {{pokemonNameWithAffix}}!",

**Review by DanStevensonx — COMMENTED** [review]
Translated from Poké Corpus to Spanish, had to change line 72 from the original Poke Corpus lines since it didnt specify the move name

**Pull request closed without merging**